docs/pages/enroll-resources/auto-discovery/databases/databases.mdx

```diff
@@ -75,14 +75,6 @@ Here's an example database discovery configuration for the Discovery Service:
 ```yaml
 discovery_service:
     enabled: true
-    # discovery_group is used to group discovered resources into different
-    # sets. This is required when you have multiple Teleport Discovery services
-    # running. It prevents discovered services from colliding in Teleport when
-    # managing discovered resources.
-    # If two Discovery Services match the same resources, they must be in the
-    # same discovery group.
-    # If two Discovery Services match different resources, they must be in
-    # different discovery groups.
```

Comment:
Why did you remove these explanations from everywhere? They seemed useful, discovery group is not an obvious concept to understand without the comment.

Reply:
I removed them but added the `
(!docs/pages/includes/discovery/discovery-group.mdx!)` partial where it was not already present.

I'm fine with either, but I think we should stick to only one form of documentation (either inline with the `yaml` code block or with the partial).